.ci/jobsDSL/template_mbp.dsl.j2

```diff
@@ -0,0 +1,173 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//pip install j2cli
+//  JOB_NAME=apm-pipeline-library-mbp REPO=apm-pipeline-library JENKINSFILE=.ci/Jenkinsfile j2 --format=env .ci/jobsDSL/template_mbp.dsl.j2
```

Comment:
Can we create a README.md in this folder regarding how to use it?

Reply:
sure thing

Reply:
documentation added